package.json

```diff
@@ -55,7 +55,6 @@
   },
   "devDependencies": {
     "@playwright/test": "^1.39.0",
-    "@swup/cli": "^5.0.1",
```

Comment:
Maybe I'm missing something -- isn't this required for the build step?

Reply:
Ooops, right!! Wait

Reply:
... I got confused because parallel plugin was not yet using @swup/cli. Re-installed it.